company/okrs/2019_q4.md

```diff
@@ -21,19 +21,19 @@
       1. VP Eng KR: 100% of monthly releases ship on time.
       1. VP Eng KR: No tier 1 or 2 customers are blocked from upgrading to a monthly release once it ships.
       1. **Distribution: Automate time-consuming ops and support tasks.**
-         1. Distribution KR: Major/minor and patch releases take 1 hour (including general QA but excluding customer-specific testing).
-         1. Distribution KR: All customer specific testing is complete within 2 business days of tagging any release.
+         1. Distribution KR: Major/minor and patch releases take 1 hour (including general QA but excluding customer-specific testing). => QA has been fully automated and releases are possible in under an hour. In practice, a 1 hour release has not yet happened due to QA finding release blockers.
```

Comment:
```suggestion
         1. Distribution KR: Major/minor and patch releases take 1 hour (including general QA but excluding customer-specific testing). => QA has been fully automated and releases are possible in under an hour. In practice, a 1 hour release has not yet happened due to QA finding release blockers. [RFC 86](https://docs.google.com/document/d/1mbnLHBhEPT2c8ttoxQaQRstivW0JIbKHKtjg-ZRr5WM/edit) addresses how to amortize QA over PRs to eliminate release blockers so that releases can take under an hour.
```